plugin/path_login.go

```diff
@@ -316,7 +316,12 @@ func (b *GcpAuthBackend) pathIamLogin(ctx context.Context, req *logical.Request,
 		return nil, errors.New("service account is empty")
 	}
 
-	alias, err := getIAMAlias(role, serviceAccount)
+	conf, err := b.config(ctx, req.Storage)
+	if err != nil {
+		return logical.ErrorResponse("unable to retrieve GCP configuration"), nil
+	}
+
+	alias, err := conf.getIAMAlias(role, serviceAccount)
```

Comment:
The conf returned on 319 can be nil if there was no entry in storage. This applies also to line 469.

Reply:
Good catch. Added a check for it and returns an error if no config found